interop_binaries/src/lib.rs

```diff
@@ -106,31 +106,9 @@ where
 #[serde(tag = "type")]
 pub enum VdafObject {
     Prio3Count,
-    Prio3CountVec {
-        length: NumberAsString<usize>,
-    },
-    Prio3Sum {
-        bits: NumberAsString<usize>,
-    },
-    Prio3SumVec {
-        bits: NumberAsString<usize>,
-        length: NumberAsString<usize>,
-    },
-    Prio3Histogram {
-        buckets: Vec<NumberAsString<u64>>,
-    },
-    #[cfg(feature = "fpvec_bounded_l2")]
-    Prio3FixedPoint16BitBoundedL2VecSum {
-        length: NumberAsString<usize>,
-    },
-    #[cfg(feature = "fpvec_bounded_l2")]
-    Prio3FixedPoint32BitBoundedL2VecSum {
-        length: NumberAsString<usize>,
-    },
-    #[cfg(feature = "fpvec_bounded_l2")]
-    Prio3FixedPoint64BitBoundedL2VecSum {
-        length: NumberAsString<usize>,
-    },
+    Prio3CountVec { length: NumberAsString<usize> },
+    Prio3Sum { bits: NumberAsString<u32> },
+    Prio3Histogram { buckets: Vec<NumberAsString<u64>> },
```

Comment:
These need to be renamed to include `Aes128` again

Reply:
I also updated `VdafInstance`; I was hoping to use the more modern names as they're shorter, but consistency is probably more important.